utils/utils.js

```diff
@@ -105,17 +103,18 @@ async function createDocumentHandler() {
 			docCompletionButton: document.getElementById('docCompletionButton').value || 'Complete DBQ'
 		}
 		
-		const doc = await zegal.createDocument(doctypePayload, options)
+		const doc = await zegal.createDocument(doctypePayload, options, 'anyModalBodyId')
```

Comment:
what's that for ?

Reply:
@Anthony-Michel we are sending div's id to zegal-client